crates/proof-of-sql-parser/src/lib.rs

```diff
@@ -36,7 +36,7 @@ pub use resource_id::ResourceId;
 // lalrpop-generated code is not clippy-compliant
 lalrpop_mod!(#[allow(clippy::all, missing_docs, clippy::missing_docs_in_private_items, clippy::pedantic)] pub sql);
 
-/// Implement Deserialize through FromStr to avoid invalid identifiers.
+/// Implement Deserialize through `FromStr` to avoid invalid identifiers.
```

Comment:
It might be hard to catch them all, but as much as possible try to find all the spots. In this case, it should be `Deserialize`, or even better [`Deserialize`].

Reply:
Would be nice if you can identify some of the ones that clippy doesn't automatically catch. That's a bit beyond the scope of the lints, so up to you how thorough you want to be here.

```suggestion
/// Implement `Deserialize` through `FromStr` to avoid invalid identifiers.
```

Or

```suggestion
/// Implement [`Deserialize`] through [`FromStr`] to avoid invalid identifiers.
```